designcompose/src/main/java/com/android/designcompose/FrameRender.kt

```diff
@@ -810,7 +810,7 @@ internal fun squooshShapeRender(
 
         // Make an appropriate paint.
         val shadowPaint = Paint().asFrameworkPaint()
-        shadowPaint.color = shadow.shadowStyle.color.toColor().toArgb()
+        shadowPaint.color = shadow.shadowStyle.color.getValue(variableState)?.toArgb() ?: 0
```

Comment:
I'm wondering if we should let it return@forEach when getValue(variableState) returns null

Reply:
The forEach is for the list of shadow fills. If we fail to get the variable value out of one, I would think we should still keep going through the list to draw the others.

Reply:
Yes. I agree. 
`return@forEach` will skip the current one and continue the next one in the forEach loop, like a `continue` in Java?

Reply:
Oh right, was confused and thought it was like `break`.

Reply:
Done